assets/src/edit-story/components/helpCenter/companion/index.js

```diff
@@ -39,12 +44,18 @@ export function Companion({ tipKey, onTipSelect, isLeftToRightTransition }) {
           {...tip}
         />
       ) : (
-        <Menu key={'menu'} transitionKey={'menu'} onTipSelect={onTipSelect} />
+        <Menu
+          read={read}
```

Comment:
I think we need to check that `read` is a the proper format here. I think what's happened to me is an edge case because i tested the API update a few weeks back, but it wouldn't hurt to safe guard. 
![Screen Shot 2021-02-05 at 8 35 07 AM](https://user-images.githubusercontent.com/10720454/107054283-27c1e400-678d-11eb-8121-9deacbff4c06.png)

![Screen Shot 2021-02-05 at 8 41 23 AM](https://user-images.githubusercontent.com/10720454/107054927-f269c600-678d-11eb-9840-5cb4da08c37d.png)